README.md

```diff
@@ -1,30 +1,18 @@
+#Hospitality Webstack
 
-     ,-----.,--.                  ,--. ,---.   ,--.,------.  ,------.
-    '  .--./|  | ,---. ,--.,--. ,-|  || o   \  |  ||  .-.  \ |  .---'
-    |  |    |  || .-. ||  ||  |' .-. |`..'  |  |  ||  |  \  :|  `--, 
-    '  '--'\|  |' '-' ''  ''  '\ `-' | .'  /   |  ||  '--'  /|  `---.
-     `-----'`--' `---'  `----'  `---'  `--'    `--'`-------' `------'
-    ----------------------------------------------------------------- 
+A web app that allows users to input information about community service hours that they have performed for UD's Hospitality department.
+The app will have a view for Students to input their hours and check on status, 
+Supervisors who will fill out a separate form to approve hours performed,
+and Internal users who will need to see all information in a simplistic form.
```

Comment:
mention that these users want to not only see information but have the ability to extract information from the data